config/common/culture/era.cwt

```diff
@@ -0,0 +1,36 @@
+types = {
+    type[era] = {
+        path = "game/common/culture/eras"
+    }
+}
+
+## cardinality = 0..inf
+era = {
+    ## cardinality = 1
+    ### Year when the era can start getting base exposure. Must be 0 or greater.
+    year = int
```

Comment:
Verify `int` or `int_value_field`. I could see this being `int` though.

Reply:
Tried it with a value field, throws errors.